packages/shell-api/src/integration.spec.ts

```diff
@@ -87,7 +86,7 @@ describe('Shell API (integration)', function() {
       { _id: 9, cust_id: 'Don Quis', ord_date: new Date('2020-03-20'), price: 55, items: [ { sku: 'carrots', qty: 5, price: 1.0 }, { sku: 'apples', qty: 10, price: 2.5 }, { sku: 'oranges', qty: 10, price: 2.5 } ], status: 'A' },
       { _id: 10, cust_id: 'Don Quis', ord_date: new Date('2020-03-23'), price: 25, items: [ { sku: 'oranges', qty: 10, price: 2.5 } ], status: 'A' }
     ]);
-    expect(res.acknowledged).to.equal(1);
+    expect(res.acknowledged).to.equal(true);
```

Comment:
The typings said `res.acknowledged` would be a boolean, the reality said it’s 1/0. I’m happy to make the other change and modify the types instead, if that’s preferable.

Reply:
where do we get the types from? The typings from `@types/mongodb` are a bit off, hopefully when the switch of driver to `ts` is done they will be  much more reliable. I'd keep the type from 'reality' anyway :)